### PR TITLE
[#1716] Add more fields to the comparison view

### LINKED ISSUE #1716: Add more feilds to the comparision view
Fields that we should add:
- [x] TRL (with tooltip as in #1717)
- [x] Resource Access Type
- [x] Resource Access Modes
- [x] Resource languages
- [x] Categorisation
- [ ] 

Moreover:
- [x]  change field name from "dedicated for" to "Target User"
- [x]  change field name from " Geographical availabilities" to "Resource availability"

### PR DESCRIPTION
**Pull request by kmarszalek**
- added more fields
- using https://docs.cyfronet.pl/display/FID/RDT+%28ERV%29+Resource+Profile as a reference point for fields labels after slack conversation with @bwilk 
Closes #1716

### CODE REVIEW

File: app/views/comparisons/show.html.haml [hunk 1]
```diff
@@ -31,49 +31,69 @@
...
         - @services.each do |service|
           %td
             - service.tag_list.sort.each do |tag|
               = link_to tag, services_path(tag: tag), class: "badge badge-light"
+      %tr
+        %th
+          = _("Technology Readiness Level")
+        - @services.each do |service|
+          %td{ "data-toggle": "tooltip", title: trl_description_text(service) }
+            = service.trl.map { |trl| trl.name.upcase }.join(", ")
```
Comment (goreck888) [inline app/views/comparisons/show.html.haml:84]: Add condition - if service.trl.present especially you call first element of an array

**Review by goreck888 — APPROVED** [review]

**Review by bwilk — APPROVED** [review]

**Comment by bwilk** [comment]
Minor issues:
1. Correct wording (from the Resource Profile) should be - Geographical Availability, Language Availability

General issues noticed, but out of scope of this development action:

1. Field order is somehow random. The page should reflect the order in the Resource Profile
2. Categorization and Domains don't represent the hierarchical structure 
   -  Scientific Domain -> Scientific Subdomain
   -  Category -> Subcategory   
3. Resource Access Type, Resource Order Type, Resource Access mode - there is at least 1 not needed. We need to verify which one is legacy and which are represented in the Resource Profile 3.0

If possible to squeeze it in the current sprint please fix this. If not we need to reflect that in the following sprint.

**Review by JanKapala — APPROVED** [review]

File: app/views/comparisons/show.html.haml [hunk 2]
```diff
@@ -31,49 +31,69 @@
...
       %tr.lightgrey-row
         %th
-          = _("Geographical availabilities")
+          = _("Resource Availability")
         - @services.each do |service|
           %td= service.geographical_availabilities.join(", ")
       %tr
         %th
-          = _("Languages")
+          = _("Languages Availability")
```
Comment (bwilk) [inline app/views/comparisons/show.html.haml:66]: see the summary comment

File: app/views/comparisons/show.html.haml [hunk 3]
```diff
@@ -31,49 +31,69 @@
...
       %tr
         %th
-          = _("Resource life cycle status")
+          = _("Resource Life Cycle Status")
         - @services.each do |service|
           - if service.life_cycle_status.present?
             %td= service.life_cycle_status.first.name
       %tr.lightgrey-row
         %th
-          = _("Geographical availabilities")
```
Comment (bwilk) [inline app/views/comparisons/show.html.haml:61]: wait - this was ok I think. Please see Resource profile.

**Review by bwilk — CHANGES_REQUESTED** [review]
Please see the comment above for summary

**Comment by kmarszalek** [comment]
Corrected:
1. Minor issue ->  Geographical Availability, Language Availability
2. General issues noticed -> field order

**Review by roksanaer — CHANGES_REQUESTED** [review]
PR is OK

**Review by bwilk — APPROVED** [review]

**Review by roksanaer — APPROVED** [review]

**Pull request merged**